client/tests/test_serving_model.py

```diff
@@ -0,0 +1,195 @@
+    "provider_source_fxt,resource_serving_client_fxt,is_local",
+    [
+        ('local_provider_and_source', 'resource_serving_clients', True),
+        ('hosted_sql_provider_and_source','resource_serving_clients', False),
+    ]
+)
+def test_registering_two_models_while_serving_features(provider_source_fxt,resource_serving_client_fxt, is_local, request):
+    provider, source = request.getfixturevalue(provider_source_fxt);
+    resource_client, serving_client = request.getfixturevalue(resource_serving_client_fxt)(is_local);
+
+    # Arranges the resources context following the Quickstart pattern
+    arrange_resources(provider, source, resource_client, is_local)
+
+    model_name_a = 'fraud_model_a';
+    model_name_b = 'fraud_model_b';
+
+    serving_client.features([("avg_transactions", "quickstart")], {"user": "C1410926"}, model=model_name_a)
+    serving_client.features([("avg_transactions", "quickstart")], {"user": "C1410926"}, model=model_name_b)
+
+    models = resource_client.list_models(is_local)
+
+    assert [model_name_a, model_name_b] == models
+
+
+@pytest.mark.parametrize(
+    "provider_source_fxt,resource_serving_client_fxt,is_local",
+    [
+        ('local_provider_and_source', 'resource_serving_clients', True),
+        ('hosted_sql_provider_and_source','resource_serving_clients', False),
+    ]
+)
+def test_registering_same_model_twice_while_serving_features(provider_source_fxt,resource_serving_client_fxt, is_local, request):
+    provider, source = request.getfixturevalue(provider_source_fxt);
+    resource_client, serving_client = request.getfixturevalue(resource_serving_client_fxt)(is_local);
+
+    # Arranges the resources context following the Quickstart pattern
+    arrange_resources(provider, source, resource_client, is_local)
+
+    model_name = 'fraud_model';
+
+    serving_client.features([("avg_transactions", "quickstart")], {"user": "C1410926"}, model=model_name)
+    serving_client.features([("avg_transactions", "quickstart")], {"user": "C1410926"}, model=model_name)
+
+    models = resource_client.list_models(is_local)
+
+    assert [model_name] == models
+
+
+@pytest.fixture(autouse=True)
+def before_and_after_each(setup_teardown):
+    setup_teardown()
+    yield
+    setup_teardown()
+
+
+def arrange_resources(provider, source, resource_client, is_local):
+    if is_local:
+        @provider.df_transformation(variant="quickstart", inputs=[("transactions", "quickstart")])
+        def average_user_transaction(transactions):
+            return transactions.groupby("CustomerID")["TransactionAmount"].mean()
+    else:
+        @provider.sql_transformation(variant="quickstart")
+        def average_user_transaction():
+            return "SELECT CustomerID as user_id, avg(TransactionAmount) as avg_transaction_amt from {{transactions.kaggle}} GROUP BY user_id"
+
+    user = ff.register_entity("user")
+    average_user_transaction.register_resources(
+        entity=user,
+        entity_column="CustomerID",
```

Comment:
Entity column would be user_id in this case

Reply:
Awww, interesting, @sdreyer. I wasn't aware this would work. The [Quickstart (Local)](https://docs.featureform.com/quickstart-local) _and_ the [Python SDK docs](https://sdk.featureform.com/register/#src.featureform.register.ColumnSourceRegistrar.register_resources) so `"CustomerID"` as the value for `entity_column`.

Should I update my docs PR with `"user_id"` for the value of `entity_column` to ensure we're consistent?